packages/features/bookings/lib/handleNewBooking.test.ts

```diff
@@ -0,0 +1,511 @@
+/**
+ * How to ensure that unmocked prisma queries aren't called?
```

Comment:
Maybe ensure calling an actual but disposable database? https://github.com/sabinadams/express_sample_app/blob/integration-tests/scripts/run-integration.sh

Reply:
For now I have ensured that through `fallbackMockImplementation`